client/my-sites/customer-home/cards/tasks/task.jsx

```diff
@@ -37,8 +39,10 @@ const Task = ( {
 	actionOnClick,
 	actionText,
 	actionUrl,
+	badgeText,
 	description,
 	illustration,
+	areSkipOptionsEnabled = true,
```

Comment:
Maybe `showSkipOptions`?

Reply:
yeh, the negative of that was slightly confusing. I changed it to `enableSkipOptions` as `showSkipOptions` seemed too close to the existing `areSkipOptionsVisible` variable